dev/user.clj

```diff
@@ -0,0 +1,20 @@
+(ns user
+  (:require
+    [buddy.core.codecs :refer [bytes->hex bytes->str]]
+    [buddy.core.codecs.base64 :as b64]
+    [cheshire.core :as json]
+    [clj-http.client :as http]
+    [clojure.java.io :as io]
+    [clojure.tools.namespace.repl :refer [refresh]]
+    [vault.client.ext.aws :as ext]
+    [vault.core :as vault])
+  (:import
+    com.amazonaws.DefaultRequest
+    com.amazonaws.Request
+    com.amazonaws.SignableRequest
+    com.amazonaws.auth.AWS4Signer
+    com.amazonaws.auth.AWSCredentials
+    com.amazonaws.auth.BasicSessionCredentials
+    com.amazonaws.auth.DefaultAWSCredentialsProviderChain
```

Comment:
Style: these should be grouped by package. What do you think of enabling stile checks as one of the CI tasks?

Reply:
Yeah hadn't wired that stuff up yet but I will include style when I do.